app/src/main/java/com/github/epfl/meili/forum/Post.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.epfl.meili.forum
+
+import android.os.Parcelable
+import android.util.Log
+import com.google.firebase.firestore.DocumentSnapshot
+import kotlinx.android.parcel.Parcelize
```

Comment:
Apparently import kotlinx.parcelize.Parcelize is better? Not sure. Cf @aman3014 's cleanup PR, more specifically the ChatMessage.kt class at https://github.com/meili-epfl/Meili/commit/47726e9413ea358860d961033fbf9c7573ba85f7  .

Reply:
Ok